dev/generate_bom.py

```diff
@@ -108,14 +117,18 @@ def init_argument_parser(cls, parser):
     """Initialize command-line arguments."""
     parser.add_argument('--base_dir', default='.', required=True,
                         help="Base directory containing the component's git repositories as subdirectories.")
+    parser.add_argument('--bom_alias', default='',
+                        help="Alias to rename the 'real' BOM as. This also sets the Spinnaker version as the alias.")
+    parser.add_argument('--changelog_output', default='',
+                        help="Output file to write the changelog to.")
     parser.add_argument('--container_builder', default='gcb',
                         help="Type of builder to use. Currently, the supported options are {'gcb', 'docker'}.")
     parser.add_argument('--docker_registry', default='',
                         help="Docker registry to push the container images to.")
-    parser.add_argument('--changelog_output', default='',
-                        help="Output file to write the changelog to.")
-    parser.add_argument('--bom_alias', default='',
-                        help="Alias to rename the 'real' BOM as. This also sets the Spinnaker version as the alias.")
+    parser.add_argument('--google_image_project', default='marketplace-spinnaker-release',
+                        help="GCE project we publish HA Spinnaker component images to.")
+    parser.add_argument('--github_org', default='https://github.com/spinnaker',
```

Comment:
I think that's the only thing that will get use immediately (i.e. forks). We can add support later if users want to use something else.